electron/main.mjs

```diff
@@ -31,6 +31,7 @@ async function startServer() {
   process.env.WORKSPACE_DIR = config.workspaceDir;
   process.env.GPTSCRIPT_GATEWAY_URL = config.gatewayUrl;
   process.env.GPTSCRIPT_OPENAPI_REVAMP = 'true';
+  process.env.GPTSCRIPT_BROWSER_SETTINGS_FILE = config.browserSettingsFile;
```

Comment:
This environment variable is read by the browser tool itself so that it knows where to look for its settings.

Reply:
Does this mean that for various tools that have settings that they will all have their own settings files?

Reply:
I suppose so, but right now the browser tool is the only tool that reads any kind of settings file for itself.